src/models/teams.rs

```diff
@@ -35,7 +35,11 @@ bitflags::bitflags! {
         const DELETE_PROJECT = 1 << 7;
         const VIEW_ANALYTICS = 1 << 8;
         const VIEW_PAYOUTS = 1 << 9;
-        const ALL = 0b1111111111;
+
+        const ADD_COLLECTION_ITEM = 1 << 10;
+        const REMOVE_COLLECTION_ITEM = 1 << 11;
+        const DELETE_COLLECTION = 1 << 12;
```

Comment:
collections aren't associated with teams so these shouldn't need ton be permissions

Reply:
Why can't a team maintain a list?